### ISSUE TITLE
[Bug] Selecting all files in a collection only downloads the last game selected

### ISSUE THREAD

**Issue by Zadeis** [rommapp/romm#1012]
**Is your feature request related to a problem? Please describe.**
The new collection feature is great! However individually downloading files in a collection, especially when there are a lot in a given collection, is tedious.

**Describe the solution you'd like**
Allow for downloading a zip file (no compression needed) of an entire custom collection.

**Describe alternatives you've considered**
Maybe allowing for automating of individual files however this would overwhelm a browser's download list/queue.

**Comment by zurdi15** [comment]
Hi! You can always use the multi-selector button (the one to the right of the filter bar) + click the `select all` option in the popup button or just shift + click on the first game and shift + click on the last one and then just hit the download button in the popup menu too!
In any case having a button of "download collection" can be easily done :)

**Comment by Zadeis** [comment]
Good to know! but unfortunately I have found a bug then. Before posting this feature suggestion I selected all roms/files by selecting the first, holding shift, selecting the last, thus now selecting all of them in the selecting and it only downloaded the last rom of that selection...

To be sure that wasn't a fluke I just tried again and got the same result. Shall I make a bug report for that if that isn't expected.

I still encourage the download all button! Ideally from a single archive file.
